include/deal.II/base/mpi.h

```diff
@@ -573,6 +595,76 @@ namespace Utilities
       internal::all_reduce(MPI_MIN, ArrayView<const T>(values, N),
                            mpi_communicator, ArrayView<T>(minima, N));
     }
+
+    template <typename T>
+    std::vector<T>
+    gather(const MPI_Comm &comm,
+           const T &object_to_send,
+           const unsigned int root_process)
+    {
+#ifndef DEAL_II_WITH_MPI
+      (void)comm;
+      std::vector<T> v(1, object_to_send);
```

Comment:
In a build without `MPI`, `root_process` is unused. Would you mark it as "used" using
```
(void) root_process
```
?